test/parallel_api/iterator/transform_iterator.pass.cpp

```diff
@@ -90,7 +90,7 @@ void test_simple_copy(size_t buffer_size)
 
     int identity = 0;
     auto& sycl_src_buf = test_base_data.get_buffer(UDTKind::eKeys);
-    auto host_source_begin = sycl_src_buf.get_access<sycl::access::mode::write>().get_pointer();
+    auto host_source_begin = sycl_src_buf.get_host_access(sycl::write).get_pointer();
```

Comment:
According to the SYCL 2020 specification, the deduction tag is named `sycl::write_only`; please fix it in all the places.

Reply:
Thanks, address it. https://github.com/oneapi-src/oneDPL/pull/789/commits/a0fedd95f9dfa6099fc8aba926a56e776ea57188